cmd/ilm-add.go

```diff
@@ -90,6 +90,26 @@ var ilmAddFlags = []cli.Flag{
 		Name:  "disable",
 		Usage: "disable the rule",
 	},
+	cli.IntFlag{
+		Name:  "abort-incomplete-uploads-days",
+		Usage: "the number of days to clean up incomplete multipart uploads",
+	},
```

Comment:
This is probably not useful @vadmeste - we can defer on adding this flag.

Reply:
> This is probably not useful @vadmeste - we can defer on adding this flag.

Why is that ? incomplete uploads causes costs for users, I think it is good to have.

Reply:
We automatically cleanup in MinIO - for AWS we don't care @vadmeste - users can set this usng import/export

Reply:
Basically, implement things in `mc` that are meaningful with the MinIO server

Reply:
> Basically, implement things in `mc` that are meaningful with the MinIO server

Done